chart/otomi/localtest.sh

```diff
@@ -1,7 +1,8 @@
 set -e
 
-export VALUES_INPUT=/tmp/otomi/values.yaml
-# export CI=1
+export VALUES_INPUT=${VALUES_INPUT:-tmp/otomi/values.yaml}
```

Comment:
this is not optional....the chart ONLY has access to `/tmp` so please don't change stuff without testing the impact

Reply:
```suggestion
export VALUES_INPUT=/tmp/otomi/values.yaml
```